server/server.go

```diff
@@ -94,6 +128,10 @@ func NewServer(config *Config) (*Server, error) {
 		restoreProgression: progress.NewProgressionWrapper(progress.ChainSyncRestore),
 	}
 
+	if logFileErr != nil {
+		m.logger.Info("Log file", "fallback to console output err", logFileErr.Error())
+	}
+
```

Comment:
As previously discussed, this shouldn't be a soft error but a fatal one.
Also, the place of checking this should be at the moment where you're initializing the logger itself

Reply:
Refactored logger instantiation fe41b1e3945aa3ff7c35ff6095786ce8d717692c